### DISCUSSION TITLE
HELP: 500 internal server error

### DISCUSSION THREAD

**Discussion by tayseerk** [community/community#67248] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, whenever I attempt to open a specific file it states there is a 500 internal server error but every other file works without any issue. Can somebody help, please?

**Comment by ghost** [comment]
me too https://github.com/torvalds/linux/blob/d8d7cd656361529c334e3b90535e81d818fc1157/include/uapi/linux/io_uring.h

**Comment by moises-silva** [comment]
For me is only .c or .h files apparently that are broken when browsing on the web. I can open files for other languages.

**Reply by tayseerk** [reply]
All my other .c files work except the one I need to submit for a class so maybe some .c files are broken and some aren't?

**Reply by moises-silva** [reply]
Yes I just started testing, and I can render some C files but not others.

**Reply by moises-silva** [reply, marked as answer]
You should be able to still clone the repo and get your file, I think is just the web view that is broken.

**Reply by tayseerk** [reply]
it worked! thank you so much

**Comment by empyreanx** [comment]
I can confirm. Opening a *.c or *.h results in a code 500

**Comment by empyreanx** [comment]
The web view is still broken

**Comment by muralivijay** [comment]
same here i can,t open particular c header file 500 error showing

**Comment by moises-silva** [comment]
It appears to be fixed now, I can open all C files I've tried.
